main/webapp/modules/core/scripts/project/history-apply-dialog.html

```diff
@@ -3,12 +3,13 @@
     <div class="dialog-header" bind="dialogHeader"></div>
     <div class="dialog-body" bind="dialogBody">
       <div class="dialog-instruction"><label for="textareaId" bind="or_proj_pasteJson"></label></div>
+      <button class="button" bind="operationJsonButton"></button>
```

Comment:
```suggestion
      <input type="file" id="file-input" bind="operationJsonButton" />
```
How about simply using a native file input instead? I think it would be more recognizeable as a file field ("Select" in itself is not very suggestive). I think it should also simplify the code (no need for a hidden input) and improve accessibility.

Reply:
Done